Utilities/MCStepLogger/include/MCStepLogger/ROOTIOUtilities.h

```diff
@@ -0,0 +1,186 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//****************************************************************************
+//* This file is free software: you can redistribute it and/or modify        *
+//* it under the terms of the GNU General Public License as published by     *
+//* the Free Software Foundation, either version 3 of the License, or        *
+//* (at your option) any later version.                                      *
+//*                                                                          *
+//* Primary Authors: Benedikt Volkel <benedikt.volkel@cern.ch>               *
+//*                                                                          *
+//* The authors make no claims about the suitability of this software for    *
+//* any purpose. It is provided "as is" without express or implied warranty. *
+//****************************************************************************
```

Comment:
This part (from line 11 to 21) seems to be non-standard (and duplicating the actual licence information further above). I would be in favour of removing ... but awaiting further opinion from @dberzano.

Reply:
Ok, I see. However, I checked the repo and I have found these lines in at least 36 files in the latest commit in `dev` as well. Depending on the decision here these files should be updated as well I guess.

Reply:
Agreed.

Reply:
In my understanding we should strictly stick to new licence information. (And cleanup leftovers where this is still in the code). So please remove these extra additions.